ui/src/org/ensRegistrar.ts

```diff
@@ -78,15 +78,30 @@ export function formatFee(fee: ethers.BigNumber): string {
   );
 }
 
-export async function commit(
+// Expire one hour from now.
+const deadline = ethers.BigNumber.from(Math.floor(Date.now() / 1000)).add(3600);
```

Comment:
This needs to be calculated when `commit` is called and not when the module is instantiated. Otherwise this lies in the past at some point.

Reply:
23a171bb6a82f75edbe1fc238dbdcaecba4130aa